README.md

```diff
@@ -1,25 +1,24 @@
-[![Build Status](https://travis-ci.org/tapanalyticstoolkit/spark-tk.svg?branch=master)](https://travis-ci.org/tapanalyticstoolkit/spark-tk)
+[![Build Status](https://travis-ci.org/trustedanalytics/spark-tk.svg?branch=master)](https://travis-ci.org/trustedanalytics/spark-tk)
```

Comment:
I think you need to update your branch.  This is showing you're reverting back to trustedanalytics, when we should now be on tapanalyticstoolkit.